src/pages/blog/baremetal-sig.md

```diff
@@ -0,0 +1,53 @@
+---
+templateKey: blog-post
+title: Bare Metal SIG at Open Infra Summit Berlin!
+author: Julia Kreger
+date: 2022-06-17T20:00:00.000Z
+category:
+  - label: Development Updates
+    id: category-A7fnZYrE9
+---
+
+Just over a week ago, the Bare Metal SIG finally had it's first in-person meeting. That whole pandemic hindered a gathering for some time.
```

Comment:
Do we usually add a space here? Just curious because there's no space between baremetal in the title.

Reply:
Eh, habit. Spacing is super weird with the translation engine :(